commands/leb128.go

```diff
@@ -0,0 +1,69 @@
+package commands
+
+import (
+	"fmt"
+	"io"
+	"strconv"
+
+	"github.com/filecoin-project/go-leb128"
+	cmdkit "github.com/ipfs/go-ipfs-cmdkit"
+	cmds "github.com/ipfs/go-ipfs-cmds"
+)
+
+var leb128Cmd = &cmds.Command{
+	Helptext: cmdkit.HelpText{
+		Tagline:          "Leb128 cli enc/decode",
+		ShortDescription: `Reading and writing leb128.`,
+	},
+	Subcommands: map[string]*cmds.Command{
+		"read":  readLeb128Cmd,
+		"write": writeLeb128Cmd,
```

Comment:
nit: I know you're just going off of my ideas in #3030 but in retrospect I'd slightly prefer `encode` and `decode` here.